app/controllers/AmendCaseSendInformationController.scala

```diff
@@ -46,6 +49,7 @@ class AmendCaseSendInformationController @Inject() (
   getData: DataRetrievalAction,
   requireData: DataRequiredAction,
   sessionRepository: SessionRepository,
+  navigator: AmendNavigator,
```

Comment:
CLARIFICATION: This is not 
val navigator.
Is that to do with the way file upload is not extending Navigation?

Reply:
It is.  The other controllers implement the `Navigation` which gives them access to handy "nextPage" and "backLink" methods.  But that only works if the controller is responsible for one "Page" and this controller has two.  So it has to use the Navigator directly.